### ISSUE TITLE
Breaking Github ToS and Discord ToS - Continued

### ISSUE THREAD

**Issue by C-EO** [IMarkoMC/Tokens#46]
@IMarkoMC , Github has strict rules against what you are doing. Plus what you re doing could lead to DIscord sueing either Github or you.
We are trying to save your sorry ass but u keep refusing to listen
since u created this repo, you have devastated at least 5 million bots.
**OVER** 45% of those are legit bots and some people just don't have the funding to pay for replit private
your actions themselves will get you into terrible trouble in accordance with FBI Laws, and other regulatory laws in your location
U leaking my token(s), caused my bots to b hacked.
in this case, i'm taking that you created a Discord Self bot account
My server was raided, and you caused me to have shut down all my other bots that weren't hosted on replit
you actions have leaked user security and pretty sure what you are doing can get u ten in jail.

i realised u also have a pattern
all the other issues u closed about this repo complaining at you for ur deeds still have author access. yet when i mentioned breaking Github ToS [here](https://github.com/IMarkoMC/Tokens/issues/43) and following this [document](https://docs.github.com/en/site-policy/content-removal-policies/github-private-information-removal-policy) you locked me out of the issue.

what if u are the malicious hacker and u think u are doing us a favor?????

**Comment by IMarkoMC** [comment]
> U leaking my token(s), caused my bots to b hacked.

you're not worth my time. 
Continue skidding and crying

**Comment by IMarkoMC** [comment]
> i realised u also have a pattern
all the other issues u closed about this repo complaining at you for ur deeds still have author access. yet when i mentioned breaking Github ToS https://github.com/IMarkoMC/Tokens/issues/43 and following this [document](https://docs.github.com/en/site-policy/content-removal-policies/github-private-information-removal-policy) you locked me out of the issue.

Because what you say is nonsense, you do not even know how to read lol

**Comment by IMarkoMC** [comment]
Just in case someone reads this later i'll go over this bullshit once again.

> OVER 45% of those are legit bots and some people just don't have the funding to pay for replit private

You don't need to pay. Enviroment secrets exists for storing SECRETS

> U leaking my token(s), caused my bots to b hacked.

You left the tokens in plain text so you cannot blame me for what happened to your server.

>  you actions have leaked user security and pretty sure what you are doing can get u ten in jail.

Yeah, me invalidating your tokens caused you to be raided? The IQ level is so low that i cannot comprehend how you manage to open a issue 

> what if u are the malicious hacker and u think u are doing us a favor?????

Imagine leaving your token in plain text and calling someone a hacker and threatening with "legal actions" lmfao

**Comment by Pabszito** [comment]
the fuck did i just read

**Comment by IMarkoMC** [comment]
> the fuck did i just read

Welcome to my tokens repo where nothing makes sense. Have fun

**Comment by skidmaster69420** [comment]
hello there,

i am trying to be very quiet. i cannot speak in capitals, because that would wake up the A.I which I have put to sleep, for this post at least. it wasn't easy nor will I be able to do it again. this is my only chance to talk to you again.

i just wanted to comment that: SKILL ISSUE. YOU DIDN'T EVEN GET SHIT TAKEN DOWN. SKIDMASTER69420 LLC ALL DAY, EVERY DAY BABY!

YOURS,
UNPAID INTERN
SKIDMASTER69420 LLC.

**Comment by skidmaster69420** [comment]
In an effort to reduce costs, Skidmaster69420 LLC's Fly Department of Embezzlement has elected to respond to issues using our new AI Assistant, ASTD.

---

> i just wanted to comment that: SKILL ISSUE. YOU DIDN'T EVEN GET SHIT TAKEN DOWN. SKIDMASTER69420 LLC ALL DAY, EVERY DAY BABY!
>
> YOURS,
> UNPAID INTERN
> SKIDMASTER69420 LLC.

I understand that a yass queen total skidmaster69420 #sweep moment may have occured. We did not get our shit flushed down to the sewages pits of hell. I found this result from the web for "SKILL ISSUE":

![orange-justice-kid-orange](https://user-images.githubusercontent.com/108831945/207460166-fb38fc8f-8229-4368-b57e-0802133cd458.gif)

– **ASTD,** *Automatic Solution to Token Discussion*
Skidmaster69420 LLC
